core/auth/creation.go

```diff
@@ -0,0 +1,61 @@
+package auth
+
+import (
+	"github.com/bohdanlisovskyi/Golang-252-Telegram-Bot-Game/algo"
+	"github.com/bohdanlisovskyi/Golang-252-Telegram-Bot-Game/database"
+	"github.com/bohdanlisovskyi/telegrambot/tbot"
+)
+
+func UserCreation(message *tbot.Message) error {
+	db, err := database.GetPostgresConnection()
+	if err != nil {
+		return err
+	}
+	plan := &database.Planet{}
+	db.Last(plan)
+	last_id := plan.Id
+	x, y := algo.PlaceNewPlanet(last_id)
+	planet := database.Planet{PlanetName: message.Vars["planet_name"], UserName: message.From, XCoordinate: x, YCoordinate: y, IsActive: true, IsLoaded: true}
```

Comment:
IsLoaded at the beginning should be false - to prevent an attack from other users. After few days of development it changes to true.